packages/material/src/controls/MaterialRadioGroupControl.tsx

```diff
@@ -0,0 +1,130 @@
+/*
+  The MIT License
+
+  Copyright (c) 2018 EclipseSource Munich
+  https://github.com/eclipsesource/jsonforms
+
+  Permission is hereby granted, free of charge, to any person obtaining a copy
+  of this software and associated documentation files (the "Software"), to deal
+  in the Software without restriction, including without limitation the rights
+  to use, copy, modify, merge, publish, distribute, sublicense, and/or sell
+  copies of the Software, and to permit persons to whom the Software is
+  furnished to do so, subject to the following conditions:
+
+  The above copyright notice and this permission notice shall be included in
+  all copies or substantial portions of the Software.
+
+  THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+  IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+  FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+  AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+  LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+  OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN
+  THE SOFTWARE.
+*/
+import * as React from 'react';
+import {
+    and,
+    computeLabel,
+    ControlElement,
+    ControlProps,
+    ControlState,
+    formatErrorMessage,
+    isDescriptionHidden,
+    isEnumControl,
+    isPlainLabel,
+    mapDispatchToControlProps,
+    mapStateToControlProps,
+    RankedTester,
+    rankWith,
+    resolveSchema,
+    schemaMatches
+} from '@jsonforms/core';
+import { connectToJsonForms, Control } from '@jsonforms/react';
+import Radio from '@material-ui/core/Radio';
+import RadioGroup from '@material-ui/core/RadioGroup';
+import {
+    FormControl,
+    FormControlLabel,
+    FormHelperText,
+    FormLabel
+} from '@material-ui/core';
+
+export class MaterialRadioGroupControl extends Control<ControlProps, ControlState> {
```

Comment:
Do we have a vanilla version of this?

Reply:
Pushed new commit containing vanilla renderer as well.